recipe/meta.yaml

```diff
@@ -13,6 +13,7 @@ source:
 
 build:
   number: 0
+  python: noarch
   script: python setup.py install --single-version-externally-managed --record=record.txt
   entry_points:
     - wheel=wheel.tool:main
```

Comment:
Oops! Missed this. 😞 Please add spaces around the `=`. Seems `conda-build` sometimes doesn't like this `=` without spaces in `entry_points` of `noarch: python` packages.